Formula/ns-3.rb

```diff
@@ -16,53 +15,86 @@ class Ns3 < Formula
   end
 
   depends_on "boost" => :build
-  depends_on "python@3.10" => [:build, :test]
+  depends_on "cmake" => :build
+  depends_on xcode: [:build, "11"]
+
+  depends_on "gsl"
+  depends_on "open-mpi"
+  depends_on "python@3.10"
 
-  uses_from_macos "libxcrypt"
   uses_from_macos "libxml2"
   uses_from_macos "sqlite"
 
+  # Clears the Python3_LIBRARIES variable. Removing `PRIVATE ${Python3_LIBRARIES}`
+  # in ns3-module-macros is not sufficient as it doesn't apply to visualizer.so.
+  on_macos do
+    patch :DATA
```

Comment:
Can we do this with an `inreplace`? We should also report this issue upstream.

Reply:
This is no longer an issue upstream. Since the switch to `cppyy` for handling bindings, the python libraries are no longer linked and shared objects are no longer created in the python packages.

Reply:
In which case the patch is better then, because that makes it a bit more obvious what went wrong. The patch comment should give some sort of indication of when it can be removed.

Sorry for flip-flopping on this!

Reply:
No problem! I've restored the patch in the latest commit.